app/src/main/java/com/jerboa/datatypes/types/Others.kt

```diff
@@ -382,6 +382,22 @@ enum class ModlogActionType {
     AdminPurgeComment,
 }
 
+/**
+ * Different show nsfw states
+ */
+enum class ShowNsfwTypes {
+
+    @SerializedName("BlurEverywhere")
+    BlurEverywhere,
+
+    @SerializedName("BlurEverywhereExceptNsfw")
+    BlurEverywhereExceptNsfw,
+
+    @SerializedName("DoNotBlur")
+    DoNotBlur;
```

Comment:
Also change the order of this from doNotBlur, BlurEverywhere, other.

This keeps the order the same as before (index order) This should allow you to simply the migration by a lot. I am honestly not sure why you were doing so much changes there. But I can't really check it atm.